src/game.cpp

```diff
@@ -7361,10 +7383,21 @@ look_around_result game::look_around( catacurses::window w_info, tripoint &cente
     look_around_result result;
     do {
         if( redraw ) {
+
```

Comment:
Erhm... I've suggested to remove empty line and you have added two empty lines instead 😄